### ISSUE TITLE
[ASK] Docker build failed at "RUN python"

### ISSUE THREAD

**Issue by r-yanyo** [microsoft/recommenders#1116]
### Description
Docker build failed at 

`RUN python recommenders/scripts/generate_conda_file.py --name base`

### In which platform does it happen?
macOS: 10.15.4
docker: 19.03.8

I also tried to build in two debian server, and failed with same error.

### Other Comments
```
% DOCKER_BUILDKIT=1 docker build -t recommenders:cpu --build-arg ENV="cpu" .
[+] Building 7.6s (11/13)
 => [internal] load build definition from Dockerfile                                                                                                                         0.0s
 => => transferring dockerfile: 37B                                                                                                                                          0.0s
 => [internal] load .dockerignore                                                                                                                                            0.0s
 => => transferring context: 2B                                                                                                                                              0.0s
 => [internal] load metadata for docker.io/library/ubuntu:18.04                                                                                                              7.2s
 => [base 1/7] FROM docker.io/library/ubuntu:18.04@sha256:3235326357dfb65f1781dbc4df3b834546d8bf914e82cce58e6e6b676e23ce8f                                                   0.0s
 => CACHED [base 2/7] WORKDIR /root                                                                                                                                          0.0s
 => CACHED [base 3/7] RUN apt-get update &&     apt-get install -y curl git wget build-essential                                                                             0.0s
 => CACHED [base 4/7] RUN wget https://github.com/Kitware/CMake/releases/download/v3.15.2/cmake-3.15.2.tar.gz -O cmake.tgz &&     mkdir cmake &&     tar xzf cmake.tgz --st  0.0s
 => CACHED [base 5/7] RUN curl https://repo.continuum.io/miniconda/Miniconda3-4.7.12.1-Linux-x86_64.sh -o anaconda.sh &&     /bin/bash anaconda.sh -b -p conda &&     rm an  0.0s
 => CACHED [base 6/7] RUN git clone --depth 1 --single-branch -b master https://github.com/microsoft/recommenders                                                            0.0s
 => CACHED [base 7/7] RUN mkdir /root/.jupyter &&     echo "c.NotebookApp.token = ''" >> /root/.jupyter/jupyter_notebook_config.py &&     echo "c.NotebookApp.ip = '0.0.0.0  0.0s
 => ERROR [cpu 1/1] RUN python recommenders/scripts/generate_conda_file.py --name base                                                                                       0.3s
------
 > [cpu 1/1] RUN python recommenders/scripts/generate_conda_file.py --name base:
#11 0.259 /bin/sh: 1: python: not found
------
failed to solve with frontend dockerfile.v0: failed to build LLB: executor failed running [/bin/sh -c python recommenders/scripts/generate_conda_file.py --name base]: runc did not terminate sucessfully
```

**Comment by gramhagen** [comment]
yes, looks like Anaconda removed the version of the miniconda we are using

you can either change the Dockerfile
```
ARG ANACONDA="https://repo.continuum.io/miniconda/Miniconda3-4.7.12.1-Linux-x86_64.sh"
to
ARG ANACONDA="https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh"
```

or specify it as a build arg in docker build command
```
--build-arg ANACONDA="https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh"
```

If you are able to test and issue a PR that would be great, otherwise I will work on it later this evening

**Comment by gramhagen** [comment]
I think this specific issue is resolved by #1120
